### ISSUE TITLE
HTML summary for TreeSequence

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tskit#872]
When working in a jupyter notebook it would be nice to have a short summary of what's in the tree sequence. Basically, we want the numbers of things: trees, individuals, nodes, edges, etc. It'd be nice to get a HTML formatted table with each of these.

**Comment by benjeffery** [comment]
Would be good to add in `nbytes` after #871, and an indication of if each table has any metadata.

A collapsed tree of ts.metadata and provenance might be nice too.

**Comment by jeromekelleher** [comment]
Sounds good.

I'm imaging using this in the jupyter-execute blocks for the docs, so I don't think it should be too long, though. Mainly a quick way of saying, "look this ts has more than 1 tree" or "look, it's got n samples" or whatever.
